lib/wit/dependency.py

```diff
@@ -43,15 +44,12 @@ def resolve_deps(self, wsroot, repo_paths, download, source_map, packages, queue
         errors = errors.copy()
         subdeps = self.package.get_dependencies()
         log.debug("Dependencies for [{}]: [{}]".format(self.name, subdeps))
-        for subdep in subdeps:
-            try:
-                subdep.load(packages, repo_paths, wsroot, download)
-            except BadSource as e:
-                errors.append(e)
-                continue
 
-            sources_conflict_check(subdep, source_map)
+        self._parallel_clone(subdeps, wsroot, repo_paths, download, errors)
 
+        for subdep in subdeps:
+            subdep.load(packages, repo_paths, wsroot, False)
```

Comment:
For readability purposes, since you've replaced original argument that had the variable name `download` with just a constant `False`, can you pass this argument as a named parameter?

```suggestion
            subdep.load(packages, repo_paths, wsroot, download=False)
```